### DISCUSSION TITLE
Default interpreter path not used by default, requires manual selection

### DISCUSSION THREAD

**Discussion by c3-kaspesi** [microsoft/vscode-discussions#172] [Extension Development QnA] [answered]
Python: Restart Language Server results in ... 
```
Command 'Python: Restart Language Server' resulted in an error (command 'python.analysis.restartLanguageServer' not found)
```

with Default Interpreter Path set to `/usr/bin/python3` this still results in the same error.  

The only way I have found to resolve this is to run the command `Python:  Select Interpreter` and select `Use Python from 'python.defaultInterpreterPath' `

Is there a way to set this by default without having to have the user manually select the command and use the UI?  I would assume that default interpreter path would mean it should use this by default.

**Comment by gjsjohnmurray** [comment]
Please report this to the authors of the extension which contributes that command to the Command palette. One way to discover its identity is to use the Start Extension Bisect command.

**Reply by c3-kaspesi** [reply]
I believe Pylance/Python extension is owned by Microsoft.  Since it is a vscode extension developed by Microsoft, this seems like the correct place?

**Reply by c3-kaspesi** [reply]
Here is the link https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance

**Reply by digitarald** [reply]
The repo is https://github.com/microsoft/vscode-python/issues/new/choose

**Comment by gjsjohnmurray** [comment, marked as answer]
Please use the Report Issue option from the Help menu and choose Extension from the first dropdown, then the contributing extension in the second dropdown.

**Reply by c3-kaspesi** [reply]
We have already reported this here https://github.com/microsoft/pylance-release/issues/2873, is that the correct place?

**Reply by kdeshc3** [reply]
Bumping up @c3-kaspesi 's comment is that the correct place?

**Reply by gjsjohnmurray** [reply]
Looks correct.
